src/main/java/core/basesyntax/GeometricFigureSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeometricFigureSupplier {
+    public GeometricFigure getRandomFigure() {
+        int index = new Random().nextInt(0, GeometricFigure.values().length);
```

Comment:
Magic numbers should be avoided. Consider defining the range for `nextInt` as a constant, e.g., `private static final int FIGURE_COUNT = GeometricFigure.values().length;` and then use `nextInt(FIGURE_COUNT)`.